### DISCUSSION TITLE
Need help with installing/config Wmbusmeters in a Docker on Synology Nas

### DISCUSSION THREAD

**Discussion by Big111156** [wmbusmeters/wmbusmeters#1199] [Q&A]
I got wmbusmeters succesfully running in a docker on a Raspberry Pi and wanted to do this also on my Synology Nas

but i was not successful... i could install the container, but now i can't find any config folder or output folder ..
what did i wrong ..

would anybody here be so kind and lead me through the installation and configuration process..
Thank you..
i'am familiar with shell commands on Linux /unix

**Comment by BIBOLV** [comment]
If you intend to use RTL-SDR then check [this](https://github.com/wmbusmeters/wmbusmeters/issues/774). If you have IMST then you can try - maybe it will work.
